media/doc/README.WINE

```diff
@@ -267,7 +267,7 @@ gdi32 -
   reactos/dll/win32/gdi32/objects/linedda.c     # Synced at 20090410
 
 iphlpapi -
-  reactos/dll/win32/iphlpapi/icmp.c # Partially synced to WineStaging-1.7.55
+  reactos/dll/win32/iphlpapi/icmp.c # Synced to WineStaging-1.7.55
```

Comment:
you forgot about https://github.com/reactos/reactos/pull/537/files#diff-d220f2ef60e74532c8e40d31e07a3d57R85

Reply:
Not really.
Made it more explicit that PR target is `icmp.c` only.

Reply:
Shouldn't these 2 be at least next to each other? If that was done intentionally, for about it.

Reply:
`README.WINE` own maintenance is out of scope of this PR.